apps/condo/domains/ticket/schema/UserTicketCommentRead.test.js

```diff
@@ -0,0 +1,168 @@
+/**
+ * Generated by `createschema ticket.UserTicketCommentRead 'user:Relationship:User:CASCADE; ticket:Relationship:Ticket:CASCADE; readResidentCommentAt:DateTimeUtc;'`
+ */
+
+const faker = require('faker')
+
+const { makeLoggedInAdminClient, makeClient, UUID_RE, DATETIME_RE } = require('@core/keystone/test.utils')
+
+const { UserTicketCommentRead, createTestUserTicketCommentRead, updateTestUserTicketCommentRead, createTestTicket } = require('@condo/domains/ticket/utils/testSchema')
+const { expectToThrowAccessDeniedErrorToObjects, expectToThrowAccessDeniedErrorToObj } = require('@condo/domains/common/utils/testSchema')
+const {
+    createTestOrganizationWithAccessToAnotherOrganization,
+    createTestOrganizationEmployeeRole,
+    updateTestOrganizationEmployee, createTestOrganization, createTestOrganizationEmployee,
+} = require('@condo/domains/organization/utils/testSchema')
+const { makeClientWithProperty, makeClientWithResidentAccessAndProperty, createTestProperty } = require('@condo/domains/property/utils/testSchema')
+const { makeClientWithResidentUser } = require('@condo/domains/user/utils/testSchema')
+const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
+
+describe('UserTicketCommentRead', () => {
+    describe('employee', () => {
+        describe('create', () => {
+            test('can create UserTicketCommentRead to comment in employee organization', async () => {
```

Comment:
```suggestion
            it('can create UserTicketCommentRead to comment in employee organization', async () => {
```

Reply:
`it` and `test` are aliases: https://jestjs.io/docs/api#testname-fn-timeout
But yes, `it` is shorter :)